test/Handler/CommentsSpec.hs

```diff
@@ -11,30 +17,32 @@ spec :: Spec
 spec = withApp $ do
     describe "GET CommentsR" $ do
         it "returns a list of comments by article" $ do
-            Entity siteId _ <- createSite
-
-            u <- createUser "1"
-            c1 <- createComment (entityKey u) siteId "1" "1" "1"
-            c2 <- createComment (entityKey u) siteId "1" "2" "2"
-            c3 <- createComment (entityKey u) siteId "2" "1" "3"
+            siteId <- runDB $ insert $ buildSite
+            user <- runDB $ createUser "1"
+            (c1, c2, c3) <- runDB $ (,,)
```

Comment:
Cool. I think it will be good to play around until we get a style we're happy with here. The stuff about `runDB`/`YesodExample` is good to know.